specs/disciplina/witness/api/witness.yaml

```diff
@@ -367,10 +462,36 @@ definitions:
             * `InvalidFormat` - Failed to deserialise one of parameters.
 
             * `BlockNotFound` - Specified block does not exist.
-            * `NotEnoughMoney` - Transaction's source account has not enough money to make this transaction.
             * `AccountNotFound` - Specified account does not exist.
             * `TransactionNotFound` - Specified transaction does not exist.
 
+  TxValidationError:
+    type: object
+    required:
+      - error
+    properties:
+      error:
+        type: string
+        enum:
+          - InvalidFormat
+          - AuthorDoesNotExist
+          - SignatureIsCorrupted
+          - NotASingletonSelfUpdate
+          - NonceMustBeIncremented
+          - PaymentMustBePositive
+          - ReceiverMustIncreaseBalance
+          - SumMustBeNonNegative
+        description: >-
+          Error type:
+            * `InvalidFormat` - Failed to deserialise one of parameters.
+            * `AuthorDoesNotExist` - Source account has never received money.
+            * `SignatureIsCorrupted` - Bad signature provided.
+            * `NotASingletonSelfUpdate` - Cannot send money to yourself more than once in transaction.
+            * `NonceMustBeIncremented` - Bad nonce for source account is used. Should be exactly 'nextNonce' returned by /accounts/{address} endpoint.
+            * `PaymentMustBePositive` - You can only decrease your funds by outcoming transaction.
+            * `ReceiverMustIncreaseBalance` - You can only increase funds of receiver by outcoming transaction.
```

Comment:
"increase _the_ funds", my bad.

Reply:
I wonder how many articles are actually missing in the spec :thinking: